pom.xml

```diff
@@ -66,140 +66,144 @@
-        <jackson.version>2.16.1</jackson.version>
+        <antlr4.version>4.13.2</antlr4.version>
+        <snakeyaml.version>2.3</snakeyaml.version>
+        <gson.version>2.11.0</gson.version>
+        <jackson.version>2.18.2</jackson.version>
         <json-path.version>2.9.0</json-path.version>
-        <json-smart.version>2.4.10</json-smart.version>
-        <accessors-smart.version>2.4.9</accessors-smart.version>
-        <asm.version>9.3</asm.version>
-        <groovy.version>4.0.22</groovy.version>
-        <freemarker.version>2.3.31</freemarker.version>
+        <json-smart.version>2.5.1</json-smart.version>
+        <accessors-smart.version>2.5.1</accessors-smart.version>
+        <asm.version>9.7.1</asm.version>
+        <groovy.version>4.0.24</groovy.version>
+        <freemarker.version>2.3.34</freemarker.version>
         <bytebuddy.version>1.15.4</bytebuddy.version>
         
         <jakarta.jakartaee-bom.version>8.0.0</jakarta.jakartaee-bom.version>
         <glassfish-jaxb.version>2.3.9</glassfish-jaxb.version>
         
         <atomikos.version>6.0.0</atomikos.version>
-        <narayana.version>5.12.7.Final</narayana.version>
-        <jboss-transaction-spi.version>7.6.1.Final</jboss-transaction-spi.version>
-        <jboss-logging.version>3.2.1.Final</jboss-logging.version>
+        <narayana.version>5.13.1.Final</narayana.version>
+        <jboss-transaction-spi.version>8.0.0.Final</jboss-transaction-spi.version>
+        <jboss-logging.version>3.6.1.Final</jboss-logging.version>
         <seata.version>2.2.0</seata.version>
         
-        <netty.version>4.1.112.Final</netty.version>
-        <bouncycastle.version>1.78.1</bouncycastle.version>
+        <netty.version>4.1.117.Final</netty.version>
+        <bouncycastle.version>1.80</bouncycastle.version>
         
-        <curator.version>5.7.0</curator.version>
-        <zookeeper.version>3.9.2</zookeeper.version>
-        <audience-annotations.version>0.12.0</audience-annotations.version>
-        <jetcd.version>0.7.7</jetcd.version>
-        <vertx.version>4.5.1</vertx.version>
+        <curator.version>5.7.1</curator.version>
+        <zookeeper.version>3.9.3</zookeeper.version>
+        <audience-annotations.version>0.15.0</audience-annotations.version>
+        <jetcd.version>0.8.4</jetcd.version>
+        <vertx.version>4.5.11</vertx.version>
         
-        <grpc.version>1.65.1</grpc.version>
-        <protobuf.version>3.21.12</protobuf.version>
+        <grpc.version>1.69.1</grpc.version>
+        <protobuf.version>4.29.3</protobuf.version>
         <okhttp.version>4.12.0</okhttp.version>
         
         <elasticjob.version>3.0.4</elasticjob.version>
         
-        <slf4j.version>1.7.36</slf4j.version>
-        <logback.version>1.2.13</logback.version>
-        <commons-logging.version>1.2</commons-logging.version>
+        <slf4j.version>2.0.16</slf4j.version>
+        <logback.version>1.5.16</logback.version>
```

Comment:
- Is this too risky? https://github.com/apache/shardingsphere-elasticjob/issues/2425 has not been closed yet, and few people realize that this will break support for Spring Boot 2.